### PR TITLE
About Us Page

### LINKED ISSUE #614: About Us Page
### Overview
Create an 'About Us' page to host information on the organization, primarily built for partner/sponsor audience, but also for individuals or potential volunteers looking to learn more about what we do. 

### Action Items

Information Needed
- [x] Bonnie to share summary blurb for the page
- [x] Bonnie to share summary on workforce development
- [x] Bonnie to share list of partners
- [ ] Bonnie will write letter

Design
- [x] Low fidelity mock-up 
- [x] Review with product 
- [x] High fidelity mock-up
- [x] Review with product 

Development
- [x] Create page based on Figma mockup
- [ ] Add to menu of HfLA site - desktop and mobile
- [ ] Before publish, ask Bonnie for letter.

### Resources/Instructions

Dashboard page: https://hackforla.org/dashboard

### PR DESCRIPTION
**Pull request by drubgrubby**
Fixes #614 

This isn't actually quite ready for a PR, as even I don't think that it's ready to merge today.  However, since @daniellex0 has already done a very comprehensive visual review, I feel like the process has begun, hence the pull request.  

Before a code review I need to finish two bits of JavaScript:
1) Get the arrows in the sticky nav to work correctly.
2) Get the "read more" for the Letter from the ED section working properly.

Additionally, we need to integrate the actual letter from the actual Executive Director.

But it's a start...

### CODE REVIEW

**Comment by daniellex0** [comment]
My first visual review of the page can be accessed on the [About Us page on Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1803%3A13692) (to the right of the final mockups)

**Comment by drubgrubby** [comment]
I have addressed the issues in from Danielle's visual review.

**Comment by drubgrubby** [comment]
@daniellex0 - I did my best to make the changes from your visual review. When you have a chance, if you can have a look and see what I missed or didn't get quite right.  I finally have the sticky navigation working, and the read-more/read-less on mobile, so you can see those.  Thanks!

**Comment by jbubar** [comment]
### Here is my visual review, 
_I will do a code review tomorrow._

_tl;dr: the page you made is absolutely stunning, and there are quite a few bugs in the tablet view(which I do not believe was specified in the figma)._
___
First off, I would like to say that I am really impressed by what you were able to accomplish with this page.

### Nitty gritty visual review:
**Desktop view:**
Absolutely stunning!
1. One nitpick thing: The margin between the text and the photo are not quite right when compared to the sigma. 
>Figma
>Screen Shot 2020-12-04 at 1 24 04 AM

>this pull request
>Screen Shot 2020-12-04 at 1 23 05 AM

**Tablet view:**
There was no tablet view in the figma, So it might need to go back to the drawing board. 
Here is a list of the busts I found

2. Between pixel sizes of 766px and about 1000px the moving menu bar on the right covers the text
Screen Shot 2020-12-04 at 1 00 31 AM

3. The "Hack for LA Platform" text looks squeezed and has strange gaps 
Screen Shot 2020-12-04 at 1 09 42 AM

4. Hack for La supports: text is overlapping
Screen Shot 2020-12-04 at 1 10 03 AM

5. Accomplishments: looks good for the most part accept for the bottom.
_And it is kinda strange that the wins card links to dean’s linkediin. Maybe we could have it link to the wins page?_
Screen Shot 2020-12-04 at 1 31 09 AM

6. Metrics: looks pretty good accept for the metrics card..
Screen Shot 2020-12-04 at 1 12 03 AM

7. Make a donation has a lot of blank white space

**MOBILE VIEW:**
8. Maybe this is above mobile view  but at about 766px the text does not fill up the whole screen and looks a bit strange in the top info.
Screen Shot 2020-12-04 at 12 57 57 AM

9. I kept wanting to close the card after I read it, but I would have to scroll all the way back up to the top of the card to close it. Maybe we consider a feature where you can have a button to close it on the bottom...

10. Wins card looks a bit strange in mobile view

I believe that all of these details can be fixed in another pull request. For now I would like to merge the pull request so that the site is live and we can split this up into many issues.

File: _sass/components/_about-us.scss [hunk 1]
```diff
@@ -0,0 +1,875 @@
...
+}
+
+.header-container--about-us h1 {
+  font-weight: 900;
+  margin-bottom: 30px;
+}
+
+.header-text--about-us {
+    font-size: 18px;
+    max-width: 500px;
```
Comment (jbubar) [inline _sass/components/_about-us.scss:17]: Do you need this max-width property? It makes it pretty strange at screen sizes larger than 530px
Screen Shot 2020-12-04 at 9 27 32 AM

**Comment by drubgrubby** [comment]
Just for future reference, this is where the animated gif for the donation section is: https://github.com/civictechindex/CTI-website-frontend/issues/60#issuecomment-673788199

File: _sass/components/_about-us.scss [hunk 2]
```diff
@@ -0,0 +1,875 @@
...
+        &:active,
+        &:focus {
+          color: $color-red;
+        }
+      }
+}
+
+// CSS Grid 
+.page-content-container-grid {
+  display: grid;
```
Comment (jbubar) [inline _sass/components/_about-us.scss:39]: Your use of css grid is mind-blowingly cool

File: _sass/components/_about-us.scss [hunk 3]
```diff
@@ -0,0 +1,875 @@
...
+    position: relative;
+    margin-bottom: 15px;
+  }
+
+
+  // Make the active link different
+  .is-active {
+    color: #333333;
+    font-weight: 700;
+    color: black !important;
```
Comment (jbubar) [inline _sass/components/_about-us.scss:99]: I heard that using !important is not always good practice. I see why you used it here. 
I think it is not turning black because `a:link, a:visited` has more priority than a single class selector `.is-active`

Screen Shot 2020-12-04 at 10 35 47 AM

maybe in a future pull request you can figure that out. 
checkout [this cool resource](https://specificity.keegan.st/)

File: _sass/components/_about-us.scss [hunk 4]
```diff
@@ -0,0 +1,875 @@
...
+
+  .sec-head-img-ed {
+    padding: 0 25 0 0;
+  }
+
+  .led-closed {
+    margin-top: 25px;
+  }
+
+  .read-more {
```
Comment (jbubar) [inline _sass/components/_about-us.scss:217]: add cursor:pointer

File: _sass/components/_about-us.scss [hunk 5]
```diff
@@ -0,0 +1,875 @@
...
+    top: -75x;
+    visibility: hidden;
+  }
+
+  .last-card {
+    margin-bottom: 2rem;
+  }
+
+  .sticky-nav {
+    grid-area: nav;
```
Comment (jbubar) [inline _sass/components/_about-us.scss:136]: again. blowing my mind

File: _sass/components/_about-us.scss [hunk 6]
```diff
@@ -0,0 +1,875 @@
...
+    border-radius: 20px;
+    margin: 30px 0 45px 0;
+    padding: 10px 0 10px 0;
+  }
+
+  .small-card p {
+    text-align: center;
+  }
+
+  .metrics-image-size {
```
Comment (jbubar) [inline _sass/components/_about-us.scss:574]: not sure why this is not working... 
Screen Shot 2020-12-04 at 10 47 37 AM

File: _sass/components/_about-us.scss [hunk 7]
```diff
@@ -0,0 +1,875 @@
...
+
+
+  .more-padding {
+    padding-top: 10px;
+  }
+
+/* Below are the breakpoints and adjustments for different size monitors */
+
+// Bigger than mobile - 480px
+@media #{$bp-mobile-up} {
```
Comment (jbubar) [inline _sass/components/_about-us.scss:638]: is this for future styling?

File: _sass/components/_about-us.scss [hunk 8]
```diff
@@ -0,0 +1,875 @@
...
+
+} // End mobile up
+
+@media only screen and (max-width: 850px) {
+
+  .stick-it {
+    right: 0;
```
Comment (jbubar) [inline _sass/components/_about-us.scss:645]: This is a good solution for the overlap problem.. maybe make the max-width a bit larger... like 1000px and add a few px to the right.... just a thought

File: _sass/components/_about-us.scss [hunk 9]
```diff
@@ -0,0 +1,875 @@
...
+
+  .read-less {
+    display: none;
+  }
+
+  .last-card {
+    margin-bottom: 150px;
+  }
+
+  .about-us-section-header {
```
Comment (jbubar) [inline _sass/components/_about-us.scss:698]: both padding and margin are computed on this one. not sure if that was on purpose

File: sticky-test.html [hunk 10]
```diff
@@ -0,0 +1,88 @@
+<!--
+---
+layout: default
+title: Sticky Test
+---
+-->
+
+<div class="sticky-container">
+    <div class="sticky-element">
+        This bit should be sticky
```
Comment (jbubar) [inline sticky-test.html]: Do we need this whole file?
Did you push this by accident?

**Review by jbubar — APPROVED** [review]
Really great code. Thanks for giving me the opportunity to review it. I learned a lot
